### PR TITLE
ON-8874 # Converted services to typescript

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: src/services/s3Submit.ts [hunk 1]
```diff
@@ -141,15 +148,18 @@ const uploadToS3 = <T>(
...
   formJson: {
     definition: FormTypes.Form
-    submission: FormTypes.FormSubmission['submission']
+    submission: SubmissionTypes.FormSubmission['submission']
     submissionTimestamp: string
     keyId?: string
     formsAppId: number
   },
-  tags: Record<string, string>,
+  tags: UnknownObject,
```
Comment (mymattcarroll) [inline src/services/s3Submit.ts:162]: This is not `<string, unknown>`, its `<string, string>`.

Comment (Zaxist) [reply src/services/s3Submit.ts:162]: The jobId can be `null | undefined` as well :/

File: src/services/AWSCognitoClient.ts [hunk 2]
```diff
@@ -463,6 +459,7 @@ function sendPostRequest(url, params, success, error) {
     .join('&')
   request.send(body)
 }
+;('prop=propVal')
```
Comment (mymattcarroll) [inline src/services/AWSCognitoClient.ts:462]: ?

Comment (Zaxist) [reply src/services/AWSCognitoClient.ts:462]: no idea

**Pull request merged**